var/spack/repos/builtin/packages/cbtf-argonavis/package.py

```diff
@@ -67,6 +67,62 @@ class CbtfArgonavis(Package):
 
     parallel = False
 
+    build_directory = 'build_cbtf_argonavis'
+
+    # We have converted from Package type to CMakePackage type for all the Krell projects.
+    # Comments from Pull Request (#4765):
+    # CMakePackage is completely different from the old Package class. Previously, you had 
+    # a single install() phase to override. Now you have 3 phases: cmake(), build(), and install(). 
+    # By default, cmake() runs cmake ... with some common arguments, which you can add to by 
+    # overriding cmake_args(). build() runs make, and install() runs make install. 
+    # So you need to add the appropriate flags to cmake_args() and remove the calls to make. 
+    # See any other CMakePackage for examples.
+    # CMakePackage is documented: 
+    # http://spack.readthedocs.io/en/latest/spack.build_systems.html?highlight= \
+    # CMakePackage#module-spack.build_systems.cmake
+
+    def build_type(self):
+        if '+debug' in self.spec:
+            return 'Debug'
+        else:
+            return 'Release'
+
+    def cmake_args(self):
+        spec = self.spec
+
+        # Look for package installation information in the cbtf and cbtf-krell
+        # prefixes
+        cmake_prefix_path = join_path(
+            spec['cbtf'].prefix) + ':' + join_path(spec['cbtf-krell'].prefix)
+
+        cmake_args = []
+        cmake_args.extend(
+            ['-DCMAKE_INSTALL_PREFIX=%s' % prefix,
```

Comment:
This is already added by default in `CMakePackage`: https://github.com/LLNL/spack/blob/develop/lib/spack/spack/build_systems/cmake.py#L111